SimplyE/Simplified-Info.plist

```diff
@@ -45,6 +45,16 @@
 	<dict>
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>lyrtech.org</key>
+			<dict>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
```

Comment:
these exceptions are quite broad and a bit concerning for security, can you document why we need this on your wiki document? Just wondering if we can restrict them a bit (maybe no subdomains?)

Reply:
this is our test LCP library, lyrtech.org contains several subdomains for different services, added the domain itself as it is under our control to avoid adding multiple subdomains. Ideally should be https, not http at all

Reply:
limited that to two subdomains